src/boot/efi/secure-boot.h

```diff
@@ -3,4 +3,8 @@
 
 #include <efi.h>
 
+#define EFI_SIG_DB_GUID \
```

Comment:
Nitpick, but the global GUID is named `EFI_GLOBAL_GUID`, which is similar in name to `EFI_GLOBAL_VARIABLE` in the spec. I think this should be `EFI_IMAGE_SECURITY_DATABASE_GUID` which is longer but is lifted from the UEFI spec and thus more recognizable.

Reply:
You are right. The kernel also uses EFI_IMAGE_SECURITY_DATABASE_GUID (https://elixir.bootlin.com/linux/latest/source/include/linux/efi.h#L341). I'll change that.